.github/workflows/update_images.yml

```diff
@@ -99,30 +51,33 @@ jobs:
                 ref: 'master'
               });
               const imageTag = `master-${sha.substring(0,8)}-stable`;
-              const { image_tag: currentImageTag } = JSON.parse(process.env[`${repo.replaceAll('-', '_').toUpperCase()}_VALUES`]);
+              const valuesPath = `charts/${repo}/values.yaml`;
```

Comment:
I'm not sure the assumption that all the repo is in the same hierarchy is a good one...

Reply:
but they are.
if the situation changes we can change the script accordingly (it's gonna be easier now that it's just a script instead of using deprecated github actions)

anyway, it was the assumption was there also before this PR so I'm merging